src/scry.cr

```diff
@@ -21,12 +21,12 @@ module Scry
 
     context = Context.new
     loop do
+      content = Request.new(STDIN).read
+      request = Message.new(content).parse
```

Comment:
Now this can crash: what if the content is an invalid/malformed json rpc request, for example a bad json that cannot be parsed, or which does not have an id for the request

Reply:
Yeah I had thought about that and I am thinking about the best way to handle this.  I think further refactoring needs to happen so we can differentiate errors that happen after we receive a valid message and before.  Really we also need to be able to test the server functionality so that needs some work as well 🤔

Reply:
True, yes it definitely need to be separated. Maybe wrap it in a begin..rescue block that only log something for now, until we work on it more thoroughly